src/Views/FirmwareView.vala

```diff
@@ -33,7 +33,7 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
         Object (
             icon_name: "application-x-firmware",
             title: _("Firmware"),
-            description: _("Firmware updates provided by device manufacturers can improve performance and fix critical security issues.")
+            description: _("Upgrade internal systems stored or running on system- and pheriperial devices issued by their makers, which is sometimes critical for performance and security.")
```

Comment:
A bit longer now than "New firmware is released by device manufacturers to improve performance, and fix security issues." but all bases are at least covered.

Reply:
```suggestion
            description: _("Upgrade internal systems stored or running on system- and peripheral devices (issued by their makers), sometimes containing critical performance and security fixes.")
```